modules/developers-guide/pages/tutorials/custom-frontend.adoc

```diff
@@ -201,16 +227,22 @@ include::example$react-index.jsx[]
 +
 [source,bash]
 ----
-mv src/custom_greeting_assets/public/index.js src/custom_greeting_assets/public/index.jsx
+mv src/custom_greeting_assets/src/index.js src/custom_greeting_assets/src/index.jsx
 ----
+. Open the default `+src/custom_greeting_assets/src/index.html+` file in a text editor, then replace the body contents with `+<div id="app"></div>+`.
++
+For example:
```

Comment:
Is something missing here?

Reply:
yeah, I accidentally added it to the wrong PR. D'oh!